phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -3471,6 +3476,12 @@ public Void call() throws Exception {
                             LOGGER.info("An instance of ConnectionQueryServices was created.");
                             openConnection();
                             hConnectionEstablished = true;
+                            boolean lastDDLTimestampValidationEnabled
```

Comment:
This is fine as of now. But any reason why populating the region servers is guarded by LAST_DDL_TIMESTAMP_VALIDATION_ENABLED?

Reply:
It was breaking some Permission related ITs. I could not look into it in detail at the time. https://github.com/apache/phoenix/pull/1666#issuecomment-1730322149

Reply:
I don't think think we can keep populating regionservers guarded by LAST_DDL_TIMESTAMP_VALIDATION_ENABLED conf for a long time. In future if some other feature wants to read this regionservers list but don't want to enable LAST_DDL_TIMESTAMP_VALIDATION_ENABLED conf property.
For time being it is fine, but lets open a jira to see why Permission related IT are failing with this change. Before we merge this feature into master branch, we will have to fix that jira.